core/objects.go

```diff
@@ -230,6 +230,9 @@ type Challenge struct {
 	// Contains the error that occurred during challenge validation, if any
 	Error *probs.ProblemDetails `json:"error,omitempty"`
 
+	// For the V2 API the "error" field is deprecated in favour of "errors".
+	Errors *probs.ProblemDetails `json:"errors,omitempty"`
```

Comment:
This is defined as an array in the spec: https://ietf-wg-acme.github.io/acme/draft-ietf-acme-acme.html#rfc.section.8

Reply:
@titanous Thanks for catching that - I've updated this PR to correct the type in 
10d39d8. Presently we'll only ever return 1 error for a challenge but it might be sensible to revisit returning multiple challenge errors at a time for V2 when we implement https://github.com/letsencrypt/boulder/issues/3247